CHANGES.md

```diff
@@ -30,6 +30,8 @@
 - Fix handling of standalone `match()` or `case()` when there is a trailing newline or a
   comment inside of the parentheses. (#2760)
 - Black now normalizes string prefix order (#2297)
+- Deprecate `--experimental-string-processing` and move the functionality under
+  `--preview` (#2789)
```

Comment:
Do we want to combine the changelog entries into one?

Reply:
No, let's keep them separate. Before the release I'd like to reorganize the changelog entries though.